src/main/java/com/indore/resources/UserResource.java

```diff
@@ -22,13 +27,12 @@
 import com.indore.services.UserService;
 
 /**
- * <File Description>.
+ * <User resource to create an entity for indexing the document, search request, getting doc by id. >.
```

Comment:
Remove `<>` and refer any class from Java library on how to write comments. Suggestion: looks for `String.java`